### PR TITLE
Add option for showing/hiding CPU and Ping status

### PR DESCRIPTION
**Pull request by dahn-d**
Add otion to menu and put a bunch of if's here and there. Also make a
small change into width calculator to account possible Ping&CPU hiding.

P.S. Who ever needs these columns btw? o_0

### CODE REVIEW

**Comment by db81** [comment]
>Who ever needs these columns btw? o_0

They're good for telling when people are lagging or catching up, which is difficult to do otherwise.

File: LuaUI/Widgets/gui_chili_deluxeplayerlist.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 function widget:GetInfo()
   return {
-    name      = "Chili Deluxe Player List - Alpha 2.02",
+    name      = "Chili Deluxe Player List - Alpha 2.03",
```
Comment (sprunk) [inline LuaUI/Widgets/gui_chili_deluxeplayerlist.lua]: Don't change the name. This resets settings.

**Comment by dahn-d** [comment]
Imo placing the widget version in the name is the worst idea ever.

**Comment by GoogleFrog** [comment]
Yes, widget version in name is the worst idea. Never change the name.

**Pull request merged**